config/config.exs

```diff
@@ -6,7 +6,7 @@ if config_env() == :test do
     tags: %{},
     enable_source_code_context: true,
     root_source_code_paths: [File.cwd!()],
-    hackney_opts: [recv_timeout: 50, pool: :sentry_pool],
+    finch_opts: [recv_timeout: 50],
```

Comment:
```suggestion
    finch_opts: [receive_timeout: 50],
```
According to [`t:Finch.request_opt()`](https://hexdocs.pm/finch/Finch.html#t:request_opt/0), this should be `:receive_timeout`